ci/authn-gcp/get_func_tokens_to_files.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash -ex
+# Script that executes several curl requests to a Google cloud function
+# to get identity tokens with different audience values and writes them to files
+
+GCP_FUNC_URL="https://${GCP_ZONE}-${GCP_PROJECT}.cloudfunctions.net/${GCP_FETCH_TOKEN_FUNCTION}"
+
+main() {
+  echo 'get_func_tokens_to_file.sh'
+  validate_pre_requisites || exit 1
+
+  # Read the identity token file
+  IDENTITY_TOKEN=$(cat $IDENTITY_TOKEN_FILE)
```

Comment:
is the IDENTITY_TOKEN_FILE the same for each audience?

Reply:
It's a token to invoke the function regardless of the audience.